package.json

```diff
@@ -11,7 +11,9 @@
     "test": "ava",
     "lint-fix": "yarn lint --fix",
     "lint": "eslint 'snippets/**/*.js'",
-    "build": "exit 0"
+    "format": "prettier --write '**/*.md' --config .prettierrc.json",
+    "build": "exit 0",
+    "prepare": "husky"
```

Comment:
I think the shop has avoided pre-commit hooks, to date. Enforcement is done by preventing PRs from merging until they pass ci.

A `prettier --check` in the `lint` script is what I would expect. For precedent, see...

https://github.com/Agoric/agoric-sdk/blob/66e4cc0f36b5ed33f1b3fa235a335757ca868b9c/package.json#L60-L63